.jenkins/build_docker_images.sh

```diff
@@ -1,11 +1,37 @@
+## build_docker_images.sh
+##
+## Bash script that builds two fv3gfs-fortran Docker images: one without and
+## and one with Serialization support enabled.  By default, the Docker image
+## is sped up by using prebuilt images for the MPI, FMS, ESMF and Serialization
+## libraries. (This behaviour can be overridden by setting BUILD_FROM_INTERMEDIATE
+## to n).
+##
+## User is required to enter a valid tag as an input argument.  This tag will
+## be appended to the two Docker images built. The Docker image with Serialization
+## support enabled will have '-serialize' appended to the final image name.
+##
+## Each newly build Docker image is pushed to the VCM Google Container Registry.
+## A compressed archive of each new Docker image is created and stored in
+## Google Storage 
+##
+##  Mark Cheeseman, VCM
+##  October 8, 2020
+
+
 #!/bin/bash
```

Comment:
I think this has to be the first line, no?

Reply:
moved